openfeature/openfeature_test.go

```diff
@@ -14,7 +14,7 @@ import (
 // The `API`, and any state it maintains SHOULD exist as a global singleton,
 // even in cases wherein multiple versions of the `API` are present at runtime.
 func TestRequirement_1_1_1(t *testing.T) {
-	defer t.Cleanup(initSingleton)
+	t.Cleanup(Shutdown)
```

Comment:
The tests are failing, presumably because t.Cleanup calls Shutdown instead of initSingleton.

Reply:
argh, sorry about that - this was again unintentional. I started from scratch.

```
git reset --hard main
$ git ls-files '*_test.go' | xargs perl -pi -e 's@defer t.Cleanup@t.Cleanup@'
```